doc/tutorial.md

```diff
@@ -10,4 +10,5 @@
 5. Create a `.env` file based on `.env.template`.
 6. Seed the database: `inv dbseed`.
 7. Run the development server: `inv dev`.
+<!-- Optional -->
 8. Install [yamlfmt](https://github.com/google/yamlfmt): `brew install yamlfmt`
```

Comment:
I think some people might read the tutorial in their browser rather than reading the raw Markdown, so I think it makes sense to change this line to:

```
8. Optional: Install [yamlfmt](https://github.com/google/yamlfmt): `brew install yamlfmt`
```